cadcdata/cadcdata/transfer.py

```diff
@@ -80,10 +80,8 @@
 PROTOCOL_HTTP_PUT = 'ivo://ivoa.net/vospace/core#httpput'
 PROTOCOL_HTTPS_GET = 'ivo://ivoa.net/vospace/core#httpsget'
 PROTOCOL_HTTPS_PUT = 'ivo://ivoa.net/vospace/core#httpsput'
-DIRECTION_PROTOCOL_MAP = {'pushToVoSpace': [PROTOCOL_HTTP_PUT,
-                                            PROTOCOL_HTTPS_PUT],
-                          'pullFromVoSpace': [PROTOCOL_HTTP_GET,
-                                              PROTOCOL_HTTPS_GET]}
+DIRECTION_PROTOCOL_MAP = {'pushToVoSpace': [PROTOCOL_HTTPS_PUT],
+                          'pullFromVoSpace': [PROTOCOL_HTTPS_GET]}
 
 # supported transfer schemes
 TRANSFER_SCHEMES = ['vos', 'ad', 'mast']
```

Comment:
Does the `cadc` scheme get translated to `vos` or does it need to be eventually added this list?

Reply:
This only affects the old `cadc-data` which is never going to get `cadc` URIs. This module is only used by the old `CadcDataClient` and both are soon to be outdated so I didn't bother updating them.